backend/api/src/main/kotlin/io/tolgee/api/v2/controllers/notification/NotificationController.kt

```diff
@@ -37,8 +38,17 @@ class NotificationController(
   @AllowApiAccess
   fun getNotifications(
     @ParameterObject pageable: Pageable,
```

Comment:
Don't forget to add @SortDefault, so we don't need to manually send it in all the test and the endpoint doesn't send different data when no sort is provided.